matcher/service/celery_tasks.py

```diff
@@ -105,3 +107,35 @@ def run_deployment(
         raise self.retry(
             exc=exc, countdown=300 * (self.request.retries + 1), max_retries=1
         )
+
+
+@celery.task(name="cancel_stale_notes", track_started=True, bind=True)
+def cancel_stale_notes(
+    self, openreview_baseurl, openreview_username, openreview_password
+):
+    print("Cancelling Stale Notes")
+    from matcher.service.server import redis_pool
+
+    redis_conn = redis.Redis(connection_pool=redis_pool)
+    config_notes = redis_conn.hgetall(name="config_notes")
+    openreview_client = openreview.Client(
```

Comment:
@carlosmondra to avoid using the super user to edit the notes, could we have a "matching" user or token? I can modify the invitation so the matching user is writer of all the configuration notes.